README.md

```diff
@@ -126,3 +127,66 @@ Example : put libvlc to `native/x86`/`native/x64`
     <VlcLib86TargetDir>native/x86</VlcLib86TargetDir>
   </PropertyGroup>
 ```
+
+## Exclude some plugins from copy
+Sometimes, you want to build a minimal package for one of these reasons:
+- Reduce your package size
+- Remove attacking surface of your software by limiting its features
+- Speed up build times : Fastest copies are those that do not occur.
+- Just because you don't need that plugin, it's cleaner not to copy it.
+
+In your csproj, you can exclude some of the plugins by including them in a
+`VlcExcludeWindowsPlugins` item group.
+That exclusion will apply to all windows builds of libvlc (x86 and x64).
+
+There is not such inclusion/exclusion mechanism for Android because libvlc is built as one monolithic library on this platform.
+
+A few things to note:
+
+- It's in an `ItemGroup`, not in a `PropertyGroup` as before
+- Even if it's an exclude, we use `Include` to choose which plugins to exclude
```

Comment:
I read that sentence 3 times before I understood it ^^

Basically what you want to say is that no matter what we do, we use the `Include` keyword from MSBuild. Conveying that might be confusing, so why not use the `Exclude` MSBuild keyword that'd match our intent?
https://msdn.microsoft.com/en-us/library/ms171453.aspx#BKMK_ExcludeAttribute
Why not use it?

Reply:
We're building a string list, in which you "include" items. It doesn't have any special meaning for MSBuild (as long as you are careful to escape the `*`). Afterwards, the `%2A` are converted into `*`, which is interpreted as msbuild.

In other words:

```
<Something Include="a;b;c" Exclude="b" />
```

will result in the list `a;c`

but

```
  <SomethingElse Include="a/%2A;b" Exclude="a/foo" />
```

will result in the list `a/*;b`, and `a/foo` won't be excluded (since that string is not in the list)